speech_assets/SampleUtterances.txt

```diff
@@ -312,53 +434,121 @@ ListenToAudioPlaylistRecent shuffle recently added audio
+ListenToSong play song {Song} by the singer {Artist}
+ListenToSong play song {Song} by the {Artist}
 ListenToSong play song {Song} by {Artist}
+ListenToSong play the song {Song}
+ListenToSong play the song {Song} by the artist {Artist}
+ListenToSong play the song {Song} by the band {Artist}
+ListenToSong play the song {Song} by the group {Artist}
+ListenToSong play the song {Song} by the singer {Artist}
+ListenToSong play the song {Song} by the {Artist}
+ListenToSong play the song {Song} by {Artist}
+Menu contextual menu
+Menu go contextual menu
+Menu go menu
+Menu go right click
+Menu menu
+Menu navigate contextual menu
+Menu navigate menu
+Menu navigate right click
+Menu open contextual menu
 Menu open menu
+Menu press contextual menu
+Menu press menu
+Menu press right click
+Menu right click
 Mute mute
 Mute set mute
 Mute set unmute
 Mute toggle mute
 Mute toggle unmute
 Mute unmute
-NewShowInquiry if there are any episodes of {Show}
-NewShowInquiry if there are any new episodes of {Show}
+NewShowInquiry do we have a new episode of {Show}
+NewShowInquiry do we have a new {Show}
+NewShowInquiry do we have any episodes of {Show}
+NewShowInquiry do we have any new episodes of {Show}
+NewShowInquiry do we have {Show}
+NewShowInquiry do you have a new episode of {Show}
+NewShowInquiry do you have a new {Show}
+NewShowInquiry do you have any episodes of {Show}
+NewShowInquiry do you have any new episodes of {Show}
+NewShowInquiry do you have {Show}
 NewShowInquiry if there are any {Show}
+NewShowInquiry if there are episodes of {Show}
+NewShowInquiry if there are new episodes of {Show}
 NewShowInquiry if there is a new episode of {Show}
 NewShowInquiry if there is a new {Show}
+NewShowInquiry if there is any {Show}
+NewShowInquiry if there is episodes of {Show}
+NewShowInquiry if there is new episodes of {Show}
 NewShowInquiry if there is {Show}
 NewShowInquiry if we have a new episode of {Show}
 NewShowInquiry if we have a new {Show}
-NewShowInquiry if we have any episodes of {Show}
-NewShowInquiry if we have any new episodes of {Show}
 NewShowInquiry if we have any {Show}
+NewShowInquiry if we have episodes of {Show}
+NewShowInquiry if we have new episodes of {Show}
 NewShowInquiry if we have {Show}
 OpenRemote open controller
 OpenRemote open controls
 OpenRemote open remote
 PageDown go page down
 PageDown navigate page down
 PageDown page down
+PageDown press page down
 PageUp go page up
 PageUp navigate page up
 PageUp page up
+PageUp press page up
 PartyMode listen to all music
 PartyMode listen to random music
 PartyMode play all music
 PartyMode play random music
 PartyMode shuffle all music
 PlayPause pause
+PlayPause pause episode
+PlayPause pause movie
+PlayPause pause playback
+PlayPause pause song
+PlayPause pause track
+PlayPause pause video
 PlayPause play
+PlayPause play episode
+PlayPause play movie
+PlayPause play playback
+PlayPause play song
+PlayPause play track
+PlayPause play video
```

Comment:
It *should* be, but since we don't know a whole lot about how Alexa *actually* chooses which one to execute, I was erring on the safe side..

Like I said, I think I tried adding these and ran into intermittent problems.  If you want to leave them in, just make sure you test very thoroughly.